tutorials/mongodb-atlas-visionapi-nodejs/index.md

```diff
@@ -0,0 +1,680 @@
+---
+title: Tutorial - Google Vision API and MongoDB Atlas on Google Cloud
+description: Learn how to use Google's Vision API and MongoDB Atlas to build a metadata database with Express server and Node.js.
+author: ksmith, arajwade
+tags: Vision API, Node.js, express, MongoDB Atlas
+date_published: 
+---
+
+## Overview
+
+In this tutorial, we will demonstrate how easy it is to use Machine Learning to gain additional insights from a batch of photos that have no prior metadata attached.  By using this workflow, we will be able to quickly build a descriptive metadata MongoDB database on Atlas that can be leveraged for a variety of business use-cases.
+
+
+### Technical Complecity
+
+Beginner
+
+### Duration
+
+45 Minutes
+
+### Objectives
+
+1.  Create and configure MongoDB Atlas Free-Tier cluster on GCP
+2.  Configuring our Google Cloud Platform Account.
+3.  Configure a Node JS application on a GCE Debian VM to use Vision API 
+    (labels,landmarks and safe search)
+4.  Kick off the batch process from any web-enabled client terminal
+5.  Verify our new metadata in both the console and in our MongoDB Atlas database
+
+## Part 1: Create and configure MongoDB Atlas Free-Tier cluster on GCP
+
+1.  Create a free account on MongoDB Atlas on www.mongodb.com/cloud/atlas
+    Click on “Try Free”at the top right if you do not have an account or “Sign In” if you already have a login. 
+
+    ![image](images/image1.png)
```

Comment:
You need to add all of your images to this Pull Request, if you look at https://github.com/GoogleCloudPlatform/community/blob/cc14586f3408559d145cd3a916c1caa9d3d16699/tutorials/mongodb-atlas-visionapi-nodejs/index.md you'll see that none of the images load.

Reply:
done